CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+- `bitmovin-player@^8.31.0` as peer dependency.
+- Configuration option to set Conviva Device Metadata
```

Comment:
```suggestion
- Possibility to set detailed Conviva Device Metadata via `ConvivaAnalyticsConfiguration.deviceMetadata`
```

If we going to deprecate the existing one please add a `### Deprecated` section.